operators/video_decoder/video_decoder_context/video_decoder_context.cpp

```diff
@@ -24,6 +24,11 @@ void VideoDecoderContext::setup(ComponentSpec& spec) {
              "async_scheduling_term",
              "Asynchronous Scheduling Condition",
              "Asynchronous Scheduling Condition");
+  spec.param(device_id_,
```

Comment:
It seems this parameter is never used in the code anywhere?

Reply:
Hi @AndreasHeumann , This is newly added parameter which is set to '0' by default and which will work for most of the cases.  This parameter can be used to specify Cuda Device ID and required to be set only incase of multiple Cuda devices and app needs to use specific Cuda device with the ID other than '0'. Hoping it answers your question, let me know otherwise, thanks.

Reply:
Thanks @apatole. I tried to find the line where `device_id_` de-referenced and used, but I could not. Could you point me to the code line?
I also noticed that that the parameter is not mentioned in the Readme.